### ISSUE TITLE
Train on my own data...

### ISSUE THREAD

**Issue by JayjieL** [machrisaa/tensorflow-vgg#66]

**Comment by JayjieL** [comment]
Hello,when i use vgg19_trainable to train my own data from scratch, i got the problem that  the loss and acuuracy on test data keep the same, it seems that the network is not working.Have you ever met such a problem?Thank you in advance for your contribution and hope to hear from you soon.

**Comment by SdustZhangzhen** [comment]
@JayjieL Hello, I also want to train my own data about Remote Sensing image. But I have no ideal about how to make the label. Can you give some suggests? Thanks. 你好，我也想用VGG19训练自己的样本数据，但是不知道怎么制作标签和训练数据，您有什么好的建议吗？谢谢！

**Comment by JayjieL** [comment]
> @JayjieL Hello, I also want to train my own data about Remote Sensing image. But I have no ideal about how to make the label. Can you give some suggests? Thanks. 你好，我也想用VGG19训练自己的样本数据，但是不知道怎么制作标签和训练数据，您有什么好的建议吗

> @JayjieL Hello, I also want to train my own data about Remote Sensing image. But I have no ideal about how to make the label. Can you give some suggests? Thanks. 你好，我也想用VGG19训练自己的样本数据，但是不知道怎么制作标签和训练数据，您有什么好的建议吗？谢谢！

File vgg19_trainable.py already provide the base network structure. So, just modify the input layer to your image size(or you can resize your pictures), and change the output dimension. As for making labels, you can manually mark them(or try to use labeled data)

**Comment by JayjieL** [comment]
> @JayjieL Hello, I also want to train my own data about Remote Sensing image. But I have no ideal about how to make the label. Can you give some suggests? Thanks. 你好，我也想用VGG19训练自己的样本数据，但是不知道怎么制作标签和训练数据，您有什么好的建议吗？谢谢！

File vgg19_trainable.py already provide the base network structure. So, just modify the input layer to your image size(or you can resize your pictures), and change the output dimension. As for making labels, you can manually mark them(or try to use labeled data)

**Comment by pancratm** [comment]
@JayjieL hello, i want to know that what does the'vgg19.npy' file mean? does it contain both image information and label information? and what kind of form should my train data be according to the 'vgg19_trainable.py'? And now I have already had a batch of images and a list of labels. I'm sorry that i'm really new to this. Thanks.

**Comment by Ruolingdeng** [comment]
Do you know how to implement the "use this to build the VGG object", Please?

**Comment by usmanxia** [comment]
@JayjieL were you able to train the model? I am exactly facing the same problem you mentioned that the loss and accuracy do not change.
Any help is really appreciated.

**Comment by JayjieL** [comment]
@usmanxia  It seems that there is an integrated interface under the Keras framework, which can be called directly.（I have not used this network anymore）
